content/en/introduction/features.md

```diff
@@ -1,125 +1,169 @@
 
 ![Overview](https://pek3b.qingstor.com/kubesphere-docs/png/20200202153355.png)
 
-The following modules elaborate the key features and benefits provided by KubeSphere container platform.
+The following modules elaborate on the key features and benefits provided by KubeSphere. For detailed information, see the respective chapter in this guide.
 
 ## Provisioning and Maintaining Kubernetes
 
-### Provisioning Kubernetes Cluster
+### Provisioning Kubernetes Clusters
 
-KubeSphere Installer allows you to deploy Kubernetes on your infrastructure out of box, provisioning Kubernetes cluster with high availability. It is recommended that at least three master nodes are configured behind a load balancer for production environment.
+KubeSphere Installer allows you to deploy Kubernetes on your infrastructure out of box, provisioning Kubernetes clusters with high availability. It is recommended that at least three master nodes are configured behind a load balancer for production environment.
 
 ### Kubernetes Resource Management
 
-KubeSphere provides graphical interface for creating and managing Kubernetes resources, including Pods and Containers, Workloads, Secrets and ConfigMaps, Services and Ingress, Jobs and CronJobs, HPA, etc. As well as powerful observability including resources monitoring, events, logging, alerting and notification.
+KubeSphere provides a graphical web console, giving users a clear view of a variety of Kubernetes resources, including Pods and containers, clusters and nodes, workloads, secrets and ConfigMaps, services and Ingress, jobs and CronJobs, and applications. With wizard user interfaces, users can easily interact with these resources for service discovery, HPA, image management, scheduling, high availability implementation, container health check and more.
+
+As KubeSphere 3.0 features enhanced observability, users are able to keep track of resources with multi-dimensional monitoring data, such as events, auditing logs, alerts and notifications.
 
 ### Cluster Upgrade and Scaling
 
-KubeSphere Installer provides ease of setup, installation, management and maintenance. Moreover, it supports rolling upgrades of Kubernetes clusters so that the cluster service is always available while being upgraded. Additionally, it provides the ability to roll back to previous stable version in case of failure. Also, you can add new nodes to a Kubernetes cluster in order to support more workloads by using KubeSphere Installer.
+KubeSphere Installer provides an easy way of installation, management and maintenance. Moreover, it supports rolling upgrades of Kubernetes clusters so that the cluster service is always available while being upgraded. Additionally, it provides the ability to roll back to the previous stable version in case of failure. Also, you can add new nodes to a Kubernetes cluster to include more workloads by using KubeSphere Installer.
```

Comment:
Remove `Additionally, it provides the ability to roll back to the previous stable version in case of failure.`